pennylane/ops/op_math/controlled.py

```diff
@@ -328,6 +328,11 @@ def hash(self):
     def has_matrix(self):
         return self.base.has_matrix
 
+    # pylint: disable=missing-function-docstring
+    @property
+    def basis(self):
+        return self.base.basis
```

Comment:
This inherits from symbolic op so you don't need this since it will be inherited anyways

Reply:
Operator versus operation thing. easier just to push the attribute up to operator.

Reply:
this can be removed, it is indeed the same as its super()